node/service/src/chain_spec/frequency.rs

```diff
@@ -14,8 +14,7 @@ use sp_core::ByteArray;
 use sp_runtime::traits::AccountIdConversion;
 
 /// Specialized `ChainSpec` for the normal parachain runtime.
-pub type ChainSpec =
-	sc_service::GenericChainSpec<frequency_runtime::RuntimeGenesisConfig, Extensions>;
+pub type ChainSpec = sc_service::GenericChainSpec<Extensions>;
```

Comment:
Yes, I can see that. The generic parameter has been removed, and adding it will cause a compilation error. You can check the details [here](https://github.com/paritytech/polkadot-sdk/pull/4410).

Reply:
Yeah, I was thinking maybe we can replace all of them with just 1 type but if it's not that easy we can ignore it.